### ISSUE TITLE
Why we contrain the table name and column name?

### ISSUE THREAD

**Issue by WingsGo** [apache/incubator-doris#3415]
**Is your feature request related to a problem? Please describe.**
I found that we do a constrain in some place such as table name and column name , and we use this regex to check `private static final String COMMON_NAME_REGEX = "^[a-zA-Z][a-zA-Z0-9_]{0,63}$"` I wonder why we need to check it and what's the influence after delete this check?

**Comment by morningman** [comment]
Mainly to prevent some strange characters and cause some unnecessary trouble.
Did you meet some problem with it?

**Comment by WingsGo** [comment]
I just wonder if we can support chinese and Is there anything that we should deal with it?

**Comment by morningman** [comment]
> I just wonder if we can support chinese and Is there anything that we should deal with it?

In fact, I don't know any additional problems. But it may need a strict test.
By the way, have you heard about [Occam's Razor](https://en.wikipedia.org/wiki/Occam%27s_razor). ^O^

**Comment by WingsGo** [comment]
> > I just wonder if we can support chinese and Is there anything that we should deal with it?
> 
> In fact, I don't know any additional problems. But it may need a strict test.
> By the way, have you heard about [Occam's Razor](https://en.wikipedia.org/wiki/Occam%27s_razor). ^O^

Yes, I konw you means, maybe we should make sure is it a real demand
